File: lecture003/patternA.cpp
#include<iostream>
using namespace std;
int main(int args,char**aggv){
    int n;
    cout<<"Enter the no of rows : ";
    cin>>n;
    int nst=1;
    int nsp=n-1;
    for(int r=0;r<n;r++){
        for(int csp=1;csp<=nsp;csp++){
            cout<<" ";
        }
        for(int cst=1;cst<=nst;cst++){
            cout<<"*";
        }
        nst+=2;
        nsp--;
        cout<<endl;
    }
}

File: lecture003/patternA.java
import java.util.Scanner;

public class patternA {
    public static Scanner scn = new Scanner(System.in);

    public static void main(String[] args) {

        int n;
        System.out.print("Enter no of rows : ");
        n = scn.nextInt();
        int nst = 1;
        int nsp = n - 1;
        for (int r = 0; r < n; r++) {
            for (int csp = 1; csp <= nsp; csp++) {
                System.out.print(" ");
            }
            for (int cst = 1; cst <= nst; cst++) {
                System.out.print("*");
            }
            nst += 2;
            nsp--;
            System.out.println();
        }
    }

}